source/client-side-encryption/tests/basic.yml

```diff
@@ -0,0 +1,53 @@
+runOn:
+  - minServerVersion: "4.1.9"
+database_name: &database_name "default"
+collection_name: &collection_name "default"
+
+# Applies to the test collection.
+data: []
+json_schema: {'properties': {'ssn': {'encrypt': {'keyId': [{'$binary': {'base64': 'AAAAAAAAAAAAAAAAAAAAAA==', 'subType': '04'}}], 'bsonType': 'string', 'algorithm': 'AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic', 'initializationVector': {'$binary': {'base64': 'aWlpaWlpaWlpaWlpaWlpaQ==', 'subType': '00'}}}}, 'randomized': {'encrypt': {'keyId': [{'$binary': {'base64': 'AAAAAAAAAAAAAAAAAAAAAA==', 'subType': '04'}}], 'bsonType': 'string', 'algorithm': 'AEAD_AES_256_CBC_HMAC_SHA_512-Randomized'}}}, 'bsonType': 'object'}
+
+# Create a separate MongoClient for your key vault.
+key_vault_data: [{key()}]
+
+tests:
+  - description: "Insert a document with auto encryption"
+    clientOptions:
+      client_side_encryption_opts:
+        kms_providers:
+          aws: {} # Credentials filled in from environment.
+        auto_encrypt_map: {}
+    operations:
+      - name: insertOne
+        collectionOptions: { auto_encrypt: { enabled: True } }
+        arguments:
+          document: &doc0 { _id: 1, ssn: "457-55-5462" }
+      - name: find
+        collectionOptions: { auto_encrypt: { enabled: True } }
+        arguments:
+          filter: { _id: 1 }
+        result: [*doc0]
+    expectations:
```

Comment:
Add the key vault query as well. This may require that we specify the database_name in the expectations, unless we put the key vault in the same database as all the other collections.

Reply:
Done in new version.